installer/upgrader/Migrations/V5/adminLangString.yaml

```diff
@@ -0,0 +1,220 @@
+langStrings:
+  -
+    value: Add Job Titles
+    unitId: add_job_titles
+  -
+    value: Job Title
+    unitId: job_title
+  -
+    value: Locations
+    unitId: locations
+  -
+    value: Number of Employees
+    unitId: number_of_employees
+  -
+    value: Add Location
+    unitId: add_location
+  -
+    value: Address
+    unitId: address
+  -
+    value: Add Location
+    unitId: add_location
+  -
+    value: Email Configuration
+    unitId: email_configuration
+  -
+    value: Mail Sent As
+    unitId: mail_sent_as
+  -
+    value: Sending Method
+    unitId: sending_method
+  -
+    value: SECURE SMTP
+    unitId: secure_smtp
+  -
+    value: SMTP
+    unitId: smtp
+  -
+    value: Sendmail
+    unitId: sendmail
+  -
+    value: Path to Sendmail
+    unitId: path_to_sendmail
+  -
+    value: SMTP Host
+    unitId: smtp_host
+  -
+    value: SMTP Port
+    unitId: smtp_port
+  -
+    value: Use SMTP Authentication
+    unitId: use_smtp_authentication
+  -
+    value: SMTP User
+    unitId: smtp_user
+  -
+    value: SMTP Password
+    unitId: smtp_password
+  -
+    value: TLS
+    unitId: tls
+  -
+    value: Optional - the mail server requires the use of TLS security.
+    unitId: optional_-_the_mail_server_requires_the_use_of_tls_security
```

Comment:
TODO:: Validate `unitId` characters in the helper class. better to allow only alpha numeric and `_`